### DISCUSSION TITLE
How do I stop address popups?

### DISCUSSION THREAD

**Discussion by geoffmozz** [osmandapp/OsmAnd#18292] [General]
Can I switch this behaviour off in OsmAnd settings?

If I leave my finger on the screen for too long it causes an address popup. I rarely, if ever, use these.

**Comment by DmitryAlexei** [comment]
Do you mean *Map context menu* (see https://osmand.net/docs/user/map/map-context-menu)? If yes, this behavior cannot be changed.

**Comment by geoffmozz** [comment]
OK. Thanks Dmitry. It is the popup Map context menu I'd like to switch off....not today though :-)
